client/setup.cfg

```diff
@@ -26,15 +26,15 @@ install_requires =
     protobuf>=3.20.0
     typeguard<3.0.0
     grpcio>=1.47.0
-    numpy==1.21.6
+    numpy
```

Comment:
Any argument to set this to a >=

Reply:
I'm not sure I'm following your comment here. Are you saying you'd prefer some `>=` specification?